velox/expression/CastExpr-inl.h

```diff
@@ -275,6 +275,30 @@ VectorPtr CastExpr::applyDecimalToIntegralCast(
   return result;
 }
 
+template <typename FromNativeType>
+VectorPtr CastExpr::applyDecimalToBooleanCast(
+    const SelectivityVector& rows,
+    const BaseVector& input,
+    exec::EvalCtx& context,
+    const TypePtr& /*fromType*/,
+    const TypePtr& toType) {
+  VectorPtr result;
+  context.ensureWritable(rows, toType, result);
```

Comment:
nit: should ensureWritable just use boolean for totype and hard code it ?

Reply:
Thanks. I fixed it.